rpcs3/Emu/Cell/lv2/sys_uart.h

```diff
@@ -1,10 +1,316 @@
 #pragma once
 
 #include "Emu/Memory/vm_ptr.h"
+#include "Utilities/mutex.h"
+#include "Utilities/cond.h"
+
+enum : u32
+{
+	PS3AV_RX_BUF_SIZE 						= 0x800,
```

Comment:
the part between the enum name and the = should all be indented with spaces instead of tabs